### PR TITLE
Aydar/task 3

### LINKED ISSUE #8: TASK-3: Profile
- ### Getting all user data (by user id)
- ### Editing user data (tg username, name, surname)
- ### Get all users usernames (for calling in comments)

### PR DESCRIPTION
**Pull request by olli2017**

### CODE REVIEW

File: src/main/java/com/star_trello/darkside/service/UserService.java [hunk 1]
```diff
@@ -1,16 +1,50 @@
...
+    UserSessionService userSessionService;
+
     public User getUserById(int id) {
         return userRepo.getUserById(id);
     }
+
+    public ResponseEntity<?> getUserByIdWrapper(int id) {
+        User user = getUserById(id);
+        if (user == null) {
+            return ResponseEntity.notFound().build();
```
Comment (avdeev1) [inline src/main/java/com/star_trello/darkside/service/UserService.java]: Are it and `ResponeEntity.status(404).build()` the same things?

File: src/main/java/com/star_trello/darkside/service/UserService.java [hunk 2]
```diff
@@ -1,16 +1,50 @@
...
+        }
+
+        return ResponseEntity.ok(user);
+    }
+
+    public ResponseEntity<?> updateUserProfile(String token, EditingProfileDto editingProfile) {
```
Comment (avdeev1) [inline src/main/java/com/star_trello/darkside/service/UserService.java:34]: Do we need `@Transactional` annotation on each method?

**Review by avdeev1 — COMMENTED** [review]
👍

File: src/main/java/com/star_trello/darkside/service/UserService.java (hunk 1 above)
Comment (olli2017) [reply src/main/java/com/star_trello/darkside/service/UserService.java]: synonim. make this in your manner(our project style)

**Review by sophiekh — APPROVED** [review]

**Review by avdeev1 — APPROVED** [review]
Myp

**Pull request merged**